docs/en/transform-v2/deduplicate.md

```diff
@@ -0,0 +1,84 @@
+# Deduplicate
+
+> Deduplicate transform plugin
+
+## Description
+
+Deduplicate rows by specified fields.
+
+## Options
+
+| name                 | type  | required | default value |
+|----------------------| ----- | -------- |---------------|
+| duplicate_fields     | array | yes      |               |
+
+### duplicate_fields [array]
+
+Duplicate rows by the field of array
+
+### common options [string]
+
+Transform plugin common parameters, please refer to [Transform Plugin](common-options.md) for details
+
+## Example
+
+The data read from source is a table like this:
+
+| id | name     | age | 
+|----|----------|-----|
+| 1  | Joy Ding | 20  | 
+| 5  | Joy Ding | 20  | 
+| 2  | Kin Dom  | 14  | 
+| 9  | Kin Dom  | 14  |
+
+The source table data must sort by duplicate fields. For example, JDBC source:
```

Comment:
The source table data must partition by duplicate fields.

Reply:
When the data is partitioned, multiple transform objects will be created to read in parallel, and the data will be partitioned first and then sorted

Reply:
It is unsupported for the partitioned data yet, unless the data partitioned by the duplicate fields first.

Reply:
> When the data is partitioned, multiple transform objects will be created to read in parallel, and the data will be partitioned first and then sorted

+1

Reply:
too restrictive